src/python/tests/appengine/libs/csp_test.py

```diff
@@ -29,10 +29,11 @@ def test_simple_policy(self):
     builder.add('connect-src', 'self', quote=True)
     builder.add('script-src', 'self', quote=True)
     builder.add('script-src', 'scripts.test.tld')
+    builder.add_sourceless('block-all-mixed-content')
```

Comment:
Shouldn't we test with upgrade-insecure-requests that is actually used in code ?

Reply:
Good catch. I initially had block-all-mixed-content in both places but didn't think to update it here.